### PR TITLE
add some CDTs for ppc64le

### PR DESCRIPTION
**Pull request by jayfurmanek**
@conda-forge/staged-recipes
FYI: @isuruf @hmaarrfk @jakirkham 

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vend other packages
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Needed for: https://github.com/conda-forge/gstreamer-feedstock, https://github.com/conda-forge/r-rgl-feedstock

### CODE REVIEW

**Comment by jayfurmanek** [comment]
@conda-forge/staged-recipes

Hi all. Can I get a review for these? These are needed for the `gstreamer` recipe, which is part of the aarch/ppc64le migrator package list.

A matching PR for the same set of CDTs for aarch64 was merged already:
https://github.com/conda-forge/staged-recipes/pull/9028

This PR is modeled after that one, except for ppc64le.

Thanks!

**Comment by isuruf** [comment]
Can you fix the linter issues?

**Comment by jayfurmanek** [comment]
Yes. Or I can try. Those two packages don't have licenses in their RPMs. The aarch64 recipe left them blank as well. I'll try and dig up the right license for them.

**Comment by jayfurmanek** [comment]
OK. That's better.

**Comment by hmaarrfk** [comment]
Why is everything in a subdirectory recipe inside the package subdirectory? I don't htink the last recipe dir is necessary and may have issues rendering maybe.

**Comment by jayfurmanek** [comment]
Mostly just convention from other forge recipes. I can change it to how yours was.

**Comment by isuruf** [comment]
For staged-recipes. `meta.yaml` should be in `recipes/libxcomposite-cos7-ppc64le` not `recipes/libxcomposite-cos7-ppc64le/recipe`

**Comment by jayfurmanek** [comment]
OK Fixed.
Needed for opencv (https://github.com/conda-forge/opencv-feedstock) as well.

File: recipes/mesa-libegl-devel-cos7-ppc64le/recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# A few variables that make this a tiny bit more portable
```
Comment (hmaarrfk) [inline recipes/mesa-libegl-devel-cos7-ppc64le/recipe/meta.yaml]: Still in a subfolder

**Comment by hmaarrfk** [comment]
Nice. I think only the egl one is missing from aarch64 now!

**Pull request merged**